pkg/plugin/installer/base.go

```diff
@@ -30,9 +30,10 @@ type base struct {
 
 func newBase(source string) base {
 	settings := cli.New()
+	pluginsDirs := filepath.SplitList(settings.PluginsDirectory)
 	return base{
 		Source:           source,
-		PluginsDirectory: settings.PluginsDirectory,
+		PluginsDirectory: pluginsDirs[0],
```

Comment:
This is almost there. If `settings.PluginsDirectory` is an empty string than calling `pluginsDir[0]` will result in a panic. Can we check the length of the slice and if it's 0 use an empty string for the value.